composer.json

```diff
@@ -20,15 +20,15 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
         "beberlei/assert": "^3.2",
         "monolog/monolog": "^2.0",
-        "nikolaposa/cascader": "^1.2",
+        "nikolaposa/cascader": "^1.3",
```

Comment:
Can you temporarily try with `1.3.*` or `1.3.0`? I'm curios if branch aliases doesn't work with `^` constraints.

Reply:
It won't make a difference. The problem is that branch aliases are equivalent to beta and alpha versions. You can't install them unless you specifically set that you allow unstable versions, or by adding the `@dev` suffix in the version constraint.

The purpose is basically that you can test versions that are not released.

I have run it locally on PHP 8 using the `^1.3@dev` constraint and it has worked.